### PR TITLE
Update PR validation script

### PR DESCRIPTION
**Pull request by Arkatufus**

### CODE REVIEW

File: .github/workflows/pr_validation.yml [hunk 1]
```diff
@@ -21,10 +21,12 @@ on:
     branches:
       - master
       - dev
+      - v1.4
```
Comment (Aaronontheweb) [inline .github/workflows/pr_validation.yml]: `v1.*` (for future-proofing)

**Review by Aaronontheweb — CHANGES_REQUESTED** [review]

File: .github/workflows/pr_validation.yml (hunk 1 above)
Comment (Arkatufus) [reply .github/workflows/pr_validation.yml]: Good idea, didn't know that this is an option

**Review by Aaronontheweb — APPROVED** [review]

**Comment by Aaronontheweb** [comment]
@Arkatufus also going to need to backport this to v1.4 :p

File: .github/workflows/pr_validation.yml [hunk 2]
```diff
@@ -51,10 +51,8 @@ jobs:
             .nuke/temp
             ~/.nuget/packages
           key: ${{ runner.os }}-${{ hashFiles('**/global.json', '**/*.csproj') }}
-      - name: Run './build.cmd All'
-        run: ./build.cmd All
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Run './build.cmd RunTests'
```
Comment (Aaronontheweb) [inline .github/workflows/pr_validation.yml:54]: So a comment here - on each PR we should run:

- Build
- Tests
- NuGet packaging

The `All` target shouldn't attempt to do a NuGet __RELEASE__ without the credentials being passed in.

We'll need to do another PR to fix this.

Comment (Arkatufus) [reply .github/workflows/pr_validation.yml:54]: Got it, i'll do a fix
